src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -1029,4 +1031,20 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
       window.scrollTo({ top: y, behavior: 'smooth' });
     }, 0);
   }
+
+  topFunction(): void {
+    window.scrollTo({ top: 0, behavior: 'smooth' });
+  }
+
+  scroll(): void {
+    const scrollToTopButton: HTMLElement | null = document.getElementById('btn-to-the-top');
```

Comment:
This function should be made private to prevent being accidentally called from outside the class.